beacon-chain/core/helpers/committee_test.go

```diff
@@ -472,7 +472,7 @@ func TestCommitteeAssignment_CantFindValidator(t *testing.T) {
 	}
 }
 
-func TestShardDelta_Ok(t *testing.T) {
+func TestShardDelta_OK(t *testing.T) {
```

Comment:
There are two TestShardDelta_OK, it was already like that, i didn't take the time to merge them.